cloudinit/sources/DataSourceOpenNebula.py

```diff
@@ -129,10 +139,10 @@ class BrokenContextDiskDir(Exception):
 
 
 class OpenNebulaNetwork(object):
-    def __init__(self, context, system_nics_by_mac=None):
+    def __init__(self, context, distro, system_nics_by_mac=None):
```

Comment:
are we making this order the default?

Reply:
"this order"?

Reply:
self, context, distro, other

Reply:
Oh, I see: `context` is an OpenNebula-specific pattern, so this shouldn't be taken as a more general statement.